images/macos/macos-10.15-Readme.md

```diff
@@ -54,30 +58,30 @@
 - GNU parallel 20200722
 - OpenSSL 1.0.2t  10 Sep 2019 `(/usr/local/opt/openssl -> /usr/local/Cellar/openssl/1.0.2t)`
 - jq 1.6
-- gpg (GnuPG) 2.2.21
+- gpg (GnuPG) 2.2.22
 - psql (PostgreSQL) 12.4
 - PostgreSQL 12.4
 - aria2 1.35.0
 - azcopy 10.6.0
 - zstd 1.4.5
-- bazel 3.4.1
+- bazel 3.5.0
 - bazelisk 1.6.1
-- helm v3.3.0+g8a4aeec
+- helm v3.3.1+g249e521
 - virtualbox 6.1.12r139181
 - mongo v4.4.0
 - mongod v4.4.0
 - Vagrant 2.2.10
 - 7-Zip 16.02
-- Newman 5.1.2
+- Newman 5.2.0
 
 
 ### Tools
-- Fastlane 2.157.2
+- Fastlane fastlane
```

Comment:
fastlane version looks broken

Reply:
it's an issue with the software generator